### PR TITLE
[WIP] no travis php 5.2 for twig 2.0

### PR DESCRIPTION
**Pull request by Tobion**
Will rebase this when #1783 is merged to master.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,13 +16,8 @@ env:
     - TWIG_EXT=yes
 
 before_script:
-    - if [ "$TWIG_EXT" == "yes" ]; then sh -c "cd ext/twig && phpize && ./configure --enable-twig && make && sudo make install"; fi
+    - if [ "$TWIG_EXT" == "yes" ]; then sh -c "cd ext/twig && phpize && ./configure --enable-twig && make && make install"; fi
```
Comment (stof) [inline .travis.yml]: this should be done in 1.x IMO, to benefit from the faster infrastructure there too

Comment (Tobion) [inline .travis.yml]: yeah was already doing that

**Comment by fabpot** [comment]
#1783 has now been merged into master

**Comment by Tobion** [comment]
You already fixed it when merging.

**Pull request closed without merging**